ios/RNSScreenStack.mm

```diff
@@ -624,16 +628,31 @@ - (BOOL)gestureRecognizerShouldBegin:(UIGestureRecognizer *)gestureRecognizer
   if (topScreen.fullScreenSwipeEnabled) {
     // we want only `RNSPanGestureRecognizer` to be able to recognize when
     // `fullScreenSwipeEnabled` is set, and we are in the bounds set by user
+#ifdef RN_FABRIC_ENABLED
+    if ([gestureRecognizer isKindOfClass:[RNSPanGestureRecognizer class]])
+#else
     if ([gestureRecognizer isKindOfClass:[RNSPanGestureRecognizer class]] &&
-        [self isInGestureResponseDistance:gestureRecognizer topScreen:topScreen]) {
+        [self isInGestureResponseDistance:gestureRecognizer topScreen:topScreen])
+#endif
+    {
       _isFullWidthSwiping = YES;
       [self cancelTouchesInParent];
       return YES;
     }
     return NO;
   }
 
-#ifndef RN_FABRIC_ENABLED
+#ifdef RN_FABRIC_ENABLED
```

Comment:
Why is it still `ifdef`?

Reply:
`customAnimationOnSwipe` is still to be implemented. I have noted this down. Resolving for now.